### DISCUSSION TITLE
Residue prediction dependencies in ESM2 – Arcadia Science

### DISCUSSION THREAD

**Discussion by giscus** [Arcadia-Science/notebook-pub-template#1] [General]
# Paired Token Prediction – Arcadia Science

https://arcadia-science.github.io/notebook-pub-template/

**Comment by ekiefl** [comment]
Welcome to this demo! You can post publicly if you sign into GitHub (you'll have to authorize Giscus). Any comment made here will be added to the Discussions page of the code repository that generated this notebook.

That means the GitHub repository is responsible for:

(1) Data and code
(2) Publication
(3) Hosting
(4) Comments

This creates a nice, self-contained ecosystem :)
